### PR TITLE
Crossbows and some entity mappings

### PR DESCRIPTION
**Pull request by Juuxel**

### CODE REVIEW

File: mappings/net/minecraft/entity/LivingEntity.mapping [hunk 1]
```diff
@@ -84,6 +84,7 @@ CLASS ahl net/minecraft/entity/LivingEntity
 	METHOD d removeStatusEffect (Lagu;)Z
 	METHOD d isPotionEffective (Lagv;)Z
 	METHOD d getCurrentExperience (Larj;)I
+	METHOD da isPullingItem ()Z
```
Comment (Juuxel) [inline mappings/net/minecraft/entity/LivingEntity.mapping]: This means right-click-holding an item (like pulling bows or using shields). There's probably a better name for this.

Comment (asiekierka) [reply mappings/net/minecraft/entity/LivingEntity.mapping]: Hmm. Right-click-holding has actions... I would consider "charging" as a term, but I wonder if there's a more generic game design-y term for this?

**Review by asiekierka — APPROVED** [review]

File: mappings/net/minecraft/entity/LivingEntity.mapping (hunk 1 above)
Comment (Juuxel) [reply mappings/net/minecraft/entity/LivingEntity.mapping]: *Charging* would work well IMO.

Comment (JamiesWhiteShirt) [reply mappings/net/minecraft/entity/LivingEntity.mapping]: This name also applies to food and potions so I think "charging" doesn't work that well.

Comment (Gegy) [reply mappings/net/minecraft/entity/LivingEntity.mapping]: `isUsingItem`? The term `use` is already used in `ItemStack#getMaxUseTime`

Comment (asiekierka) [reply mappings/net/minecraft/entity/LivingEntity.mapping]: Use sounds a tad generic to me, but might work...

Comment (Juuxel) [reply mappings/net/minecraft/entity/LivingEntity.mapping]: I changed it to `isUsingItem`.

**Review by Gegy — APPROVED** [review]

**Pull request merged**